tests/drivers/pwm/gpio_loopback/testcase.yaml

```diff
@@ -24,34 +24,34 @@ tests:
     integration_platforms:
       - nrf54h20dk/nrf54h20/cpuapp
     extra_args:
-      DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_fast_p7_0.overlay"
-      CONFIG_TEST_PWM_PERIOD_USEC=10000
 
```

Comment:
Empty lines are in wrong place. Shall be before test configuration (or remove if it makes problems).
See
https://github.com/nrfconnect/sdk-nrf/blob/60c47c50d039d17f5ae2d0acf9a6fc9d7df3b5b3/tests/drivers/pwm/gpio_loopback/testcase.yaml#L27